proxy/hello.go

```diff
@@ -0,0 +1,43 @@
+package proxy
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+// TODO: remove hello world handler
```

Comment:
Is it worth removing all the hello world code?

Reply:
I left it there intentionally, so that we can reuse that when we create our handler. There is already a ticket in the backlog to remove the unneeded code (DIS-392).